website/source/docs/install/upgrade-to-0.5.html.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: "install"
+page_title: "Upgrading to Vault 0.5"
+sidebar_current: "docs-install-upgrade-to-0.5"
+description: |-
+  Learn how to upgrade to Vault 0.5
+---
+
+# Overview
+
+This page contains the breaking changes for Vault 0.5. Please read it carefully.
```

Comment:
@pearkes That's already in the changelog which will be linked to from the same place as here. I figure that pointing to the changelog as the short list and this as the exhaustive list should be fine?